lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -613,12 +613,12 @@ def test_single_problem(self, _get_current_task):
             metadata={'graded': True},
             display_name='Problem Vertical'
         )
-        self.define_option_problem(u'Pröblem1', parent=vertical)
```

Comment:
You guys probably followed up on this offline, but to record it here for posterity...

@staubina and I talked about this in great detail, and AJ spent a long time trying to preserve the unicode characters. But the test code is mocking things out in ways that don't allow unicode characters to work properly with the code changes in this PR (the mocking is not representative of what actually happens in production). And from my memory of when Dan Friedman put these unicode characters in, it wasn't because of an actual bug, but instead because he was just trying to aggressively use unicode characters in tests to attempt to flush out issues proactively. Which is great, except that the mocking in the tests (specifically around construction of course keys) doesn't represent reality....